src/main/java/com/sourcegraph/cody/CodyToolWindowContent.java

```diff
@@ -522,7 +538,7 @@ private void sendMessage(
 
     String displayText = XmlStringUtil.escapeString(message);
     ChatMessage humanMessage = new ChatMessage(Speaker.HUMAN, message, displayText);
-    addMessageToChat(humanMessage);
+    addMessageToChat(humanMessage, true);
```

Comment:
I don't know Sourcegrph code standards, but maybe when using boolean value from java we could add a variable name?

`addMessageToChat(humanMessage, /* shouldDisplayBlinkingCursor = */ true)`

In the long term just `true` or `false` is hard to read and reason about.

Reply:
good point, thanks

Reply:
> In the long term just true or false is hard to read and reason about.

+1

However, isn't this problem solved by inlay parameter names in IDE? Also, I'm not sure if the IDE will take such a comment into account during refactoring variable name.

IMO the most *clean* approach would be to split this into two methods (smth like addMessage and addBlinkingMessage).

Reply:
I moved a few classes to Kotlin - as we disscussed it in the other PR, it's much better approach :D